docs/_static/logo.svg

```diff
@@ -0,0 +1,55 @@
+      <path sodipodi:nodetypes="cssssc" id="path3168" d="M 9.7775511,7.8115681 C 8.4505441,9.785815 8.2593681,15.587286 7.569061,19.259165 C 6.6879182,23.946139 10.865787,28.708627 11.777552,31.105686 C 13.601081,35.899804 23.483434,35.34098 23.895199,30.517451 C 24.101081,28.105686 20.408165,24.026633 17.274943,19.516518 C 14.14172,15.006403 11.568191,10.065227 9.7775511,7.8115681 z " style="fill:url(#linearGradient3332);fill-opacity:1;fill-rule:evenodd;stroke:url(#linearGradient3334);stroke-width:3;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"/>
+      <path transform="translate(-2.1048009,-0.423725)" d="M 38.705883 27.17647 A 12 12.352942 0 1 1  14.705883,27.17647 A 12 12.352942 0 1 1  38.705883 27.17647 z" sodipodi:ry="12.352942" sodipodi:rx="12" sodipodi:cy="27.17647" sodipodi:cx="26.705883" id="path2160" style="fill:#ffff00;fill-opacity:1;stroke:url(#linearGradient3310);stroke-width:3;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" sodipodi:type="arc"/>
+      <path style="fill:url(#linearGradient3326);fill-opacity:1;fill-rule:evenodd;stroke:url(#linearGradient3328);stroke-width:3;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" d="M 4.1304936,16.16451 C 2.8034861,18.138757 3.0240743,23.910816 2.5102378,27.553283 C 1.9773839,31.330562 5.1010814,33.41451 6.012846,35.811569 C 7.836376,40.605687 19.483435,41.458628 19.8952,36.635099 C 20.101082,34.223334 17.055225,30.820751 13.510238,27.457695 C 9.9652511,24.094639 5.9211338,18.418168 4.1304936,16.16451 z " id="path3156" sodipodi:nodetypes="cssssc"/>
+      <path sodipodi:nodetypes="cssssc" id="path3232" d="M 43.869506,16.16451 C 45.196513,18.138757 44.975925,23.910816 45.489762,27.553283 C 46.022616,31.330562 42.898918,33.41451 41.987154,35.811569 C 40.163624,40.605687 28.516564,41.458628 28.104799,36.635099 C 27.898917,34.223334 30.944774,30.820751 34.489761,27.457695 C 38.034748,24.094639 42.078866,18.418168 43.869506,16.16451 z " style="fill:url(#linearGradient3287);fill-opacity:1;fill-rule:evenodd;stroke:url(#linearGradient3289);stroke-width:3;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"/>
+      <path sodipodi:nodetypes="cssssc" id="path2162" d="M 23.895199,20.752745 C 22.568192,22.726992 21.112309,23.64611 18.451414,25.729754 C 15.300944,28.196766 15.178276,36.780379 16.483434,38.98804 C 19.24814,43.664511 29.483434,43.929216 31.659905,39.458628 C 32.719433,37.282301 32.59065,28.15039 29.21612,25.516519 C 25.171133,22.359345 25.685839,23.006404 23.895199,20.752745 z " style="fill:url(#linearGradient3321);fill-opacity:1;fill-rule:evenodd;stroke:url(#linearGradient3323);stroke-width:3;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"/>
+    </g>
+  </g>
+</svg>
```

Comment:
This file appears to be identical to the one at ```bodhi/server/static/bodhi-logo.svg``` - should we use a symlink instead of copying it?

Reply:
I don't know why I copied it before, but I have made it so